src/playground/public/assets/js/app.ts

```diff
@@ -29,12 +29,18 @@ async function printAskVersion(elemId: string, askScriptServerUrl: string) {
 function registerAskScriptEditor(
   editorElementId: string,
   runElementId: string,
-  askScriptServerUrl: string
+  askScriptServerUrl: string,
+  dirty: boolean
```

Comment:
Looks like `dirty` should a variable local to this function rather than the argument

Reply:
remove the flag from arguments